hkm/static/hkm/js/hkm-v2.js

```diff
@@ -337,7 +343,50 @@ palikka
       });
     }
   }
+})
+.define('app.reset-password', ['jQuery', 'docReady'], function() {
+  $('#password-reset-email').on('submit', function(event) {
+    event.preventDefault();
 
+    $.post('/', {
+      action: 'password_reset',
+      csrfmiddlewaretoken: $('input[name="csrfmiddlewaretoken"]').val(),
+      email: $(this).find('input[name="email"]').val(),
+    }).done(function () {
+        $('#password-reset-form').addClass('hidden');
+        $('#password-reset-response-success').removeClass('hidden');
+    }).fail(function () {
+      $('#password-reset-form').addClass('hidden');
+      $('#password-reset-response-error').removeClass('hidden');
+    })
+  })
+
+  // Logic for opening the password reset modal automatically
+  const url = window.location.pathname;
+  const passwordChange = url.search('reset') > -1
```

Comment:
```suggestion
  const passwordChange = url.includes('reset')
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes

I would consider naming `passwordChange` into something that suggests that it's a boolean value, such as `showPasswordChangeModal` or something to that degree.